fvtest/threadtest/abortTest.cpp

```diff
@@ -204,15 +206,17 @@ TEST(ThreadAbortTest, Waiting)
 
 	omrthread_abort(t);
 
-	omrthread_monitor_wait(testdata.exitSync);
+	if (0 == testdata.aborted) {
+		omrthread_monitor_wait(testdata.exitSync);
```

Comment:
Sorry, perhaps I haven't fully digested everything in this test, but what is this trying to achieve?

Reply:
This ensures that if the other thread already did the notify (which it should not be able to) this thread will not wait indefinitely.

Reply:
OK, thanks.